Source/Krypton Components/Krypton.Toolkit/Palette Base/PaletteMode.cs

```diff
@@ -89,17 +90,19 @@ public enum PaletteMode
         /// </summary>
         Office2007BlackDarkMode,
 
-        // ToDo: Re-enable when the gray themes are completed
-        /*
-         /// <summary>
-        /// Specifies the dark Gray color variant of the Office 2010 appearance.
+        /* ToDo: Re-enable when the gray themes are completed
```

Comment:
These commented out elements need to be moved to the end,
Because if my designer has selected 24, and then later on I have a nuget package update, then 24 will be a different number !!!
Or, The enums have explicit numbers, that are stored in the ComboBox Item tag, which are related in a Map, and they never change, regardles of commented out, OR filtered out mebers

Reply:
I left the disabled entries in their place because the dictionary has them listed in the same order. If one is re-enabled again then it's in the right spot at once. 

Keeping PaletteMode and the Dictionary in the same order it will make it easier to maintain consistency.

In the future PaletteMode will not be used as an index anymore. Like you pointed out the needed values can be retrieved from the BiDirectional map in either way.

PaletteMode will then only serve as a unique identifier.  That can be different per theme throughout builds.

That all will be implemented in the follow-up on this where all three selectors will be revised.

I kept this brief so the reporter can move on.